src/preprint/models.py

```diff
@@ -3,24 +3,221 @@
 __license__ = "AGPL v3"
 __maintainer__ = "Birkbeck Centre for Technology and Publishing"
 
+import os
 
 from django.db import models
 from django.utils import timezone
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+
+from core.file_system import JanewayFileSystemStorage
+
+
+def html_input_types():
+    return (
+        ('text', 'Text'),
+        ('select', 'Dropdown'),
+        ('checkbox', 'Checkbox'),
+        ('number', 'Number'),
+        ('date', 'Date'),
+    )
+
+def width_choices():
+    return (
+        (3, '3'),
+        (6, '6'),
+        (9, '9'),
+        (12, '12'),
+    )
+
+
+fs_path = os.path.join(settings.BASE_DIR, 'files')
```

Comment:
Should probably be a relative path to avoid generating migrations between different installs:
`fs_path = 'files/preprints'`